native/src/routes/Pois.tsx

```diff
@@ -154,14 +187,6 @@ const Pois = ({
     return <Failure code={fromError(error)} />
   }
 
-  const featureLocations = pois
-    .map(poi => poi.featureLocation)
-    .filter((feature): feature is Feature<Point> => !!feature)
-
-  const currentFeature: Feature<Point> | undefined = featureLocations.find(
-    feature => feature.properties?.id === Number(route.params.selectedPoiId)
-  )
```

Comment:
Why did you move this to the hook?

Reply:
because i set the current selected feature in the lifecycle hook? Why should i call it somewhere else?